src/components/filesystem/index.js

```diff
@@ -331,25 +332,43 @@ module.exports = {
                 that.currentDir = file.get();
                 that.updateFiles(refreshSharedWith);
             }).exceptionally(function(throwable) {
-                throwable.printStackTrace();
+                console.log(throwable.getMessage());
```

Comment:
any reason to not also do a stack trace?

Reply:
stacktrace not available in js

Reply:
I've seen java stacktraces in the browser console?

Reply:
There is no printStackTrace() on the throwable object returned at least. Seems it isn't exposed. We don't call printStackTrace anywhere in the vanilla js code.